ftd/src/interpreter/test.rs

```diff
@@ -258,3 +258,41 @@ fn evalexpr_test() {
         }
             */
 }
+
+#[test]
+fn test_extract_kwargs() {
+    let doc = interpret_helper(
+        "foo",
+        r#"-- component foo:
+    kw-args data:
+
+    -- ftd.text: Hello world
+
+    -- end: foo
+
+    -- foo:
+    bar: Hello
+    baz: World
+"#,
+    )
+    .unwrap_or_else(|e| panic!("{:?}", e));
```

Comment:
This is also wrong. What is the code end user is supposed to write, write that code. Is interpret_helper in your opinion the method we should expose?

Reply:
Point is not to test the "public API" for getting data. You have built a feature to extract data, just show me how am I supposed to use it.